### ISSUE TITLE
Remove Medihud/Securihud from tools available to borgs

### ISSUE THREAD

**Issue by Fevix** [VOREStation/VOREStation#1262]
With the "Set-Sensor-Augmentation" verb, these tools are no longer necessary for their purpose. 

Admittedly, there's no strong argument for their removal other than "No longer needed to see securihud/medihud info", so if there's some reason FOR them to remain, then by all means disregard this.

**Comment by Wickedtemp** [comment]
I've never played as a Borg before, so I'll ask. What do you see using the "suit sensor augmentation"? Just suit sensors, or is the MedHUD/SecHUD, along with its icons, included?

**Comment by Fevix** [comment]
As far as I'm aware, it's exactly the same. 

When you use the verb, it brings up a menu with 3 options: None, Security, and Medical. As far as I've been able to tell, there is zero difference between the Security option and SecHUD or the Medical option and MedHUD. 

Additionally, the verb and all three options are available regardless of which module you're using, so you could have a miner cyborg with SecHUD engaged via Set-Sensor-Augmentation

**Comment by Very-Soft** [comment]
The medihid and the set sensor augmentation mode for medical are not the same. The medihud allows you to see full info for all people, while the set sensors augmentation only allows you to see the information available from their suit sensors.

Tl;dr: if someone's suit sensors are off, you can't see any info with Set Sensor Augmentation's medical setting. Where you can with the medihud.

**Comment by Fevix** [comment]
I've seen every single person's info when using the sensor augmentation. I don't know if I just happened to come across only people with their sensors on, and am not dismissing that possibility

**Comment by Funce** [comment]
The sensor augmentation only works (ForMedHUD) if they have their suit sensors on. No Suit Sensors, No go.

**Comment by Shophaune** [comment]
Issue can be closed. Items mentioned are not available to borgs that I can tell, and their functionality has indeed been replaced by the sensor augmentation settings
